README.md

```diff
@@ -51,7 +51,7 @@ For more information on how to perform typical tasks, see the How to guides sect
 
 ### Deployment
 
-The Kafka and ZooKeeper operators can both be deployed as follows:
+The Apache Kafka and Apache ZooKeeper operators can both be deployed as follows:
```

Comment:
```suggestion
Charmed Apache Kafka and Charmed Apache ZooKeeper can both be deployed as follows:
```

Reply:
if you really want to use operator, i have noticed that elsewhere you were capitalizing that. Anyhow I would just drop operators (unless we want to only refer to the charm code), but here for instance we refer to both operator code and the workload, so I'd say that we deploy Charmed Apache Kafka and Charmed Apache ZooKeeper

Reply:
It's not that I specifically wanted to use that term. It was here long before me and out of scope for this PR.  But you're right, it does feel much better with the solution context instead. Good catch!